# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/moz.build
# Commit: fdf5dd8ff807
# Full Hash: fdf5dd8ff80792a48fd00771850104df655cd866
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-11-14 21:46:28
# Regressor Bug: 1172394
# File Overlap Count: 1
# Description:
#   Bug 1172394 - Refactor how DecodedStream is set up. r=padenot
#   
#   This patch removes the responsibility of js-facing MediaStreamTracks from the
#   MediaDecoder stack, and moves the machinery for setting up DecodedStream to
#   higher order functions like state mirroring and watchables.
# ==============================================================================

diff -r 9114318b6493 -r fdf5dd8ff807 dom/media/mediasink/moz.build
--- a/dom/media/mediasink/moz.build	Wed Nov 13 22:40:09 2019 +0000
+++ b/dom/media/mediasink/moz.build	Wed Nov 13 22:40:07 2019 +0000
@@ -8,7 +8,6 @@
     'AudioSink.cpp',
     'AudioSinkWrapper.cpp',
     'DecodedStream.cpp',
-    'OutputStreamManager.cpp',
     'VideoSink.cpp',
 ]
 
